android/src/main/java/com/getcapacitor/community/admob/rewarded/AdRewardExecutor.java

```diff
@@ -39,6 +40,16 @@ public void prepareRewardVideoAd(final PluginCall call, BiConsumer<String, JSObj
                     try {
                         final AdRequest adRequest = RequestHelper.createRequest(adOptions);
                         final String id = AdViewIdHelper.getFinalAdId(adOptions, adRequest, logTag, contextSupplier.get());
+
+                        if (call.getString("userId", "") != "" || call.getString("customData", "") != "") {
+                            final ServerSideVerificationOptions ssvOptions = new ServerSideVerificationOptions.Builder()
+                                    .setUserId(call.getString("userId", ""))
+                                    .setCustomData(call.getString("customData", ""))
```

Comment:
Here we already know `userId` and `customData` are defined so we can just use the `adOptions` values 

```
 final ServerSideVerificationOptions ssvOptions = new ServerSideVerificationOptions.Builder()
                                    .setUserId(adOptions.userId)
                                    .setCustomData(adOptions.customData)
```